test/harvestFi-strat-migration.js

```diff
@@ -0,0 +1,88 @@
+const hre = require('hardhat')
+const { describe, it } = require('mocha')
+const { ethers } = require('hardhat')
+const { expect } = require('chai')
+
+const VAULT = '0x2dCdCA085af2E258654e47204e483127E0D8b277'
+const GUEST_LIST = '0xcB16133a37Ef19F90C570B426292BDcca185BF47'
+const INVERSE_DEPLOYER = '0x3FcB35a1CbFB6007f9BC638D388958Bc4550cB28'
+const FDAI = '0xe85c8581e60d7cd32bbfd86303d2a4fa6a951dac'
+const DEPOSITOR = '0x3c7d43de93Eb6598172f20Feb8ecFE8DEc70FE41' // has lots of tokens
+const TIMELOCK = '0x446Ce1FcFF45F2A9066B7E2B4198db48e79e1bf0'
+const CTOKEN_STRAT = '0x42bE0F4FA30980f8261325E9C8Fc9CBd08dE9D6C'
```

Comment:
This should be changed to the address of the vault we're currently testing migration from. The one at `VAULT` is ETH vault with a Yearn strat, not ctoken.

Also there's a different cTokenStrat deployment per vault, this one is no longer used.